pom.xml

```diff
@@ -183,7 +183,7 @@
         <java-ipv6.version>0.17</java-ipv6.version>
         <javassist.version>3.29.1-GA</javassist.version>
         <restlet.version>2.6.0</restlet.version>
-        <slf4j.version>1.7.36</slf4j.version>
+        <slf4j.version>2.0.17</slf4j.version>
```

Comment:
This change won't suffice. The variable is used only to define NOP logger, but not the API version. You need to add slf4j-api to dependencyManagement section.

Reply:
Ok, after checking on the newest wrensec-commons snapshot, the only place with old SLF4J API is the embedded Wren:DS. That should not cause any issues IMO.